echopype/echodata/combine.py

```diff
@@ -118,6 +118,7 @@ def combine_echodata(echodatas: List[EchoData], combine_attrs="override") -> Ech
             )
         elif sonar_model is None:
             sonar_model = echodata.sonar_model
+            result.sonar_model = sonar_model
```

Comment:
in #437 this was at the end of `combine_echodata`. Which one should we use?

Reply:
Would be a good idea to merge `dev` to make sure other changes are in here also.

Reply:
I deleted mine, I forgot this was fixed already